Library/Formula/kafka.rb

```diff
@@ -14,7 +14,7 @@ class Kafka < Formula
 
   depends_on "gradle"
   depends_on "zookeeper"
-  depends_on :java => "1.7+"
+  depends_on :java => "1.7"
```

Comment:
It's better to leave a comment. Otherwise, someone(like me :smirk: ) would open PR to fix the inconsistent between `depends_on :java => "1.7"` and `Language::Java.java_home_env("1.7+")`.

@mikemcquaid It's more like `["1.7", :build]` and `["1.7+", :run]` which looks strange.